lib/ansible/modules/web_infrastructure/ansible_tower/tower_credential.py

```diff
@@ -289,13 +296,14 @@ def main():
                 if os.path.isdir(filename):
                     module.fail_json(msg='attempted to read contents of directory: %s' % filename)
                 with open(filename, 'rb') as f:
-                    module.params['ssh_key_data'] = f.read()
-
-            for key in ('authorize', 'authorize_password', 'client', 'secret',
-                        'tenant', 'subscription', 'domain', 'become_method',
-                        'become_username', 'become_password', 'vault_password',
-                        'project', 'host', 'username', 'password',
-                        'ssh_key_data', 'ssh_key_unlock'):
+                    module.params['ssh_key_data'] = f.read().decode('utf-8')
```

Comment:
@wwitzel3 @AlanCoding Need to decode this for it to work properly in py3

Reply:
@ryanpetrello Rather than using `.decode`, use `to_text()` which does the right thing for depending on the Python version.